devenv/lib/colima.py

```diff
@@ -41,17 +40,42 @@ def uninstall(binroot: str) -> None:
             pass
 
 
-def install(
-    version: str, url: str, sha256: str, reporoot: Optional[str] = ""
-) -> None:
-    if reporoot:
-        binroot = fs.ensure_binroot(reporoot)
-    else:
-        # compatibility with devenv <= 1.4.0
-        binroot = f"{root}/bin"
-        os.makedirs(binroot, exist_ok=True)
+def install_shim(binroot: str) -> None:
+    fs.write_script(
+        f"{binroot}/colima",
+        """#!/bin/sh
+export COLIMA_HOME="{COLIMA_HOME}"
+
+# needed to ensure COLIMA_HOME is what we want it to be
+unset XDG_CONFIG_HOME
+
+# no one is going to type 'devenv colima start'
+# but it's very important that colima is started with the
+# right parameters
+for arg in "$@"; do
+  if [ "$arg" = "start" ]; then
+    exec "{py}" -P -m devenv colima start
+  fi
+done
```

Comment:
this seems (1) kinda surprising and (2) pretty fragile (what if a random unrelated command also has `start` in it?)

I wouldn't expect `colima` to do anything but actual colima personally -- I realize this is a sharp edge but I think this type of hack is more likely to bite us than it is to help us -- instead maybe we should work on detecting the broken case and healing it?

Reply:
i'll walk this part back, but i have yet to find a way to change the vm's parameters like file mounts after first-start - will look again into if that's possible then